x/mongo/driver/topology/server.go

```diff
@@ -54,39 +74,13 @@ func (ss *SelectedServer) Description() description.SelectedServer {
 	}
 }
 
-// These constants represent the connection states of a server.
-const (
-	disconnected int64 = iota
-	disconnecting
-	connected
-	connecting
-	initialized
```

Comment:
Were `connecting` and `initialized` unused?

Reply:
`connecting` was only used by a `Topology` and `initialized` was only used by a `connection.